core/src/main/scala/kafka/api/ApiVersion.scala

```diff
@@ -307,4 +307,6 @@ object ApiVersionValidator extends Validator {
       case e: IllegalArgumentException => throw new ConfigException(name, value.toString, e.getMessage)
     }
   }
+
+  override def toString: String = "[" + ApiVersion.allVersions.map(_.shortVersion).distinct.mkString(", ") + "]"
```

Comment:
We're using the short version here, but it seems the default value is printed using the full version (e.g. "2.1-IV2" in the image you provided). It's a bit annoying, but I'm tempted to say we should just include all full versions since they are all valid. Either that, or we could just do all short versions plus the default. What do you think?

Reply:
Well, following the default value's format would be less confusing.